### PR TITLE
[zh-cn] Update services-networking/_index.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/_index.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Kubernetes 网络解决四方面的问题：
 
 - 一个 Pod 中的容器之间[通过本地回路（loopback）通信](/zh-cn/docs/concepts/services-networking/dns-pod-service/)。
 - 集群网络在不同 Pod 之间提供通信。
-- [Service 资源](/zh-cn/docs/concepts/services-networking/service/)允许你
+- [Service](/zh-cn/docs/concepts/services-networking/service/) API 允许你
   [向外暴露 Pod 中运行的应用](/zh-cn/docs/concepts/services-networking/connect-applications-service/)，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/_index.md:105]:
```suggestion
- [Service](/zh-cn/docs/concepts/services-networking/service/) API
  允许你[向外暴露 Pod 中运行的应用](/zh-cn/docs/concepts/services-networking/connect-applications-service/)，
```

File: content/zh-cn/docs/concepts/services-networking/_index.md [hunk 2]
```diff
@@ -112,6 +112,11 @@ Kubernetes 网络解决四方面的问题：
...
 [Cluster Networking](/docs/concepts/cluster-administration/networking/) explains how to set
 up networking for your cluster, and also provides an overview of the technologies involved.
+
+The [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) 
+tutorial lets you learn about Services and Kubernetes networking with a hands-on example.
 -->
 [集群网络](/zh-cn/docs/concepts/cluster-administration/networking/)解释了如何为集群设置网络，
 还概述了所涉及的技术。
+[连接应用和服务](/zh-cn/docs/tutorials/services/connect-applications-service/)
+的教程通过一个实际示例让您了解Services和Kubernetes网络。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/_index.md:122]:
```suggestion
[连接应用和服务](/zh-cn/docs/tutorials/services/connect-applications-service/)教程通过一个实际示例让你了解
Service 和 Kubernetes 如何联网。
```
Note: That url doesn't have a localized page yet.

**Review by windsonsea — COMMENTED** [review]
You may need to check:
https://kubernetes.io/zh-cn/docs/contribute/localization_zh/

File: content/zh-cn/docs/concepts/services-networking/_index.md [hunk 3]
```diff
@@ -101,17 +101,22 @@ Kubernetes 网络解决四方面的问题：
...
   来[发布仅供集群内部使用的服务](/zh-cn/docs/concepts/services-networking/service-traffic-policy/)。
 
 <!--
+The [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/)
+tutorial lets you learn about Services and Kubernetes networking with a hands-on example.
+
 [Cluster Networking](/docs/concepts/cluster-administration/networking/) explains how to set
 up networking for your cluster, and also provides an overview of the technologies involved.
 -->
+[连接应用和服务](/zh-cn/docs/tutorials/services/connect-applications-service/)教程通过一个实际示例让你了解
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/_index.md:119]: 这一变更的前提是对应的页面也被迁移。

**Review by howieyuen — COMMENTED** [review]
forget to update URL

**Review by howieyuen — APPROVED** [review]
/approve

**Comment by windsonsea** [comment]
/lgtm
thanks.

**Pull request merged**